### DISCUSSION TITLE
Just want to know: How to Download image in tauri app to my local machine

### DISCUSSION THREAD

**Discussion by devhimanshuhc** [tauri-apps/tauri#8840] [Q&A]
Hello everyone, I'm currently working on a project where I'm rendering images in the frontend fetched from the backend as image links.  How to download these images onto my local machine. Could anyone provide guidance or suggest a solution for achieving this?

**Comment by vonPB** [comment]
Is there anything specific about images or the format, or are you just generally asking how to store files? 
There are function to interact with the filesystem from the js api listed over here https://tauri.app/v1/api/js/fs#writebinaryfile.
If you want to store them via rust, you can use std::fs to interact with the local computers files.

**Reply by devhimanshuhc** [reply]
> Is there anything specific about images or the format, or are you just generally asking how to store files? 


@vonPB the image im getting are of two types depends on the api call. their response types are array buffer and imge link. im rendering these images in the frontend. so i am having a download button that will download that image. so for that only i wanted to know.

**Comment by devhimanshuhc** [comment]
>Is there anything specific about images or the format, or are you just generally asking how to store files?

@vonPB the image im getting are of two types depends on the api call. their response types are array buffer and imge link. im rendering these images in the frontend. so i am having a download button that will download that image. so for that only i wanted to know.
